packages/backend/src/modules/match/finalize-match.mutation.ts

```diff
@@ -0,0 +1,31 @@
+import { Arg, Field, ID, Mutation, ObjectType, Resolver } from 'type-graphql';
+import { getRepository } from 'typeorm';
+
+import DBMatch from '../../entities/match';
+
+import GQLMatch from './match';
+
+@ObjectType()
+class ReportMatchFinlaizedMutationPayload {
+	@Field()
+	match: GQLMatch;
+}
+
+@Resolver()
+export default class ReportMatchFinlaizedMutationResolver {
```

Comment:
spelling

Reply:
Same above

Reply:
Its very nice of you two to point it out but trivial stuff like this I would rater you just commit the change instead of commenting here and making someone else do it. This also means that reviews might become stale after changes. There seem to not be any functional changes that are commented, only my meme names and questions but in the future I would do big appreciate if you just commit stuff like this